README.md

```diff
@@ -228,16 +228,16 @@ For example:
 chmod 400 id_rsa && \
 ssh-add $(pwd)/id_rsa
 
-ssh vcap@<concourse_ip>
+ssh vcap@<deployer_concourse_ip>
```

Comment:
Since 48db141d73c7222bf07866e6a58b023de64aaa8d the hostname points at an ELB instead of the instance's EIP. We had a similar problem with the instructions in 8e03fd1aeb5e118d2679a83c1f2938b76b9fcd49. We could consider adding TCP/22 to the ELB and removing the EIP (is it still needed?).